app/dashboard/templates/bulk_payout_bounty.html

```diff
@@ -215,7 +223,36 @@ <h5>{% trans 'Payout Preview' %}</h5>
       .entry.active{
         background-color: #bbb;
       }
-
+      .overageAlert {
+        background-color: #E66700;
+        color: #FFF;
+        border-radius: 5px;
+        margin-top: 15px;
+        margin-bottom: 5px;
+        display: none;
+        padding-left: 15px;
+      }
+      .overageAlert > div:first-child {
+        width: 25px;
+        float: left;
+        padding-right: 10px;
+      }
+      .overageAlert > div:last-child {
+        width: calc(100% - 40px);
+        float: left;
+        margin-left: 15px;
+        padding-right: 15px;
+        padding-top: 6.5px;
+      }
+      .overageAlert > div:first-child > img {
+        width: 17px;
+        margin-top: 10px;
+      }
+      .overagePreview {
+        color: #E66700;
+        padding-bottom: 5px;
+        display: none;
+      }
```

Comment:
Looks like this is piling up @Anish-Agnihotri 
Could we push all the css into a new file  ? ^_^

Reply:
For sure, makes sense!